src/js/modules/Tile.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-static';
+import ChevronCircleRightSVG from 'js/svg/ChevronCircleRight';
+
+
+
+const Tile = ({ url, text, style, tag }) => (
+  <Link className="coa-Tile" to={url}>
+    <span className="coa-Tile__tag">{tag}</span>
+    <p className="coa-Tile__text">{text}</p>
+    <ChevronCircleRightSVG size="22"/>
+  </Link>
+);
+
+Tile.propTypes = {
```

Comment:
A+ on using prop types ✅  But, McKinley asked a good question that lead us to find that React-Static doesn't acknowledge proptypes -- or if it does, doesn't throw any warnings when proptypes are not used. Not something to worry about now, but might be something we think about when we come up with a build plan that includes testing.

Reply:
good catch McKinley. but, damn that's frustrating though. I'm over React-Static. I've also noticed that it suppresses the basic linting we used to get with create-react-app.

Reply:
also down to add this as an issue to the backlog.